test/start-stop.make

```diff
@@ -40,6 +40,11 @@ start: _work/clear-kvm.img _work/kube-clear-kvm _work/start-clear-kvm _work/ssh-
 		touch _work/clear-kvm.secretsdone; \
 	fi
 	_work/ssh-clear-kvm kubectl version --short | grep 'Server Version' | sed -e 's/.*: v\([0-9]*\)\.\([0-9]*\)\..*/\1.\2/' >_work/clear-kvm-kubernetes.version
+	if [ -e deploy/kubernetes-$$(cat _work/clear-kvm-kubernetes.version)/csinodeinfo-crd.yaml ]; then \
+	   	if ! _work/ssh-clear-kvm kubectl get crd/csinodeinfos.csi.storage.k8s.io  >/dev/null 2>&1; then \
+			_work/ssh-clear-kvm kubectl create -f - <deploy/kubernetes-$$(cat _work/clear-kvm-kubernetes.version)/csinodeinfo-crd.yaml; \
+		fi \
```

Comment:
This is more suitable for `test/setup-clear-kvm.sh`. Also squash it into the previous commit.

Reply:
Added cds installation to this script.